components/bms_boss/src/CANIO_componentSpecific.c

```diff
@@ -57,7 +57,7 @@ uint8_t CANIO_tx_getNLG513ControlByte(void)
     uint8_t ret = 0x00;
     switch (SYS.contacts)
     {
-        case SYS_CONTACTORS_CLOSED:
+        case SYS_CONTACTORS_PRECHARGE:
             ret = 0x40;
```

Comment:
magic numbers, you should add an enum or some #defines for these instead so people can understand what they mean just from reading